reinforcement_learning/rl_unity_ray/rl_unity_ray.ipynb

```diff
@@ -252,12 +252,7 @@
     "ray_version = \"0.8.5\"\n",
     "suffix = \"py36\"\n",
     "\n",
-    "image_name = \"462105765813.dkr.ecr.{region}.amazonaws.com/sagemaker-rl-ray-container:ray-{ray_version}-tf-{cpu_or_gpu}-{suffix}\".format(\n",
-    "    region=sage_session.boto_region_name,\n",
-    "    ray_version=ray_version,\n",
-    "    cpu_or_gpu=cpu_or_gpu,\n",
-    "    suffix=suffix,\n",
-    ")\n",
+    "image_name = <YOUR_IMAGE_URL_HERE>\n",
```

Comment:
Could we add a description about this as the first cell in all of these notebooks? That way a customer has more detail about why the notebook does not work out of the box

Reply:
Added for all subfolder readme and notebooks. I wanted to make sure even the examples that do not directly include specific URLs contain the warning as all of them call RLEstimator and uses those containers indirectly.